x-pack/plugins/stack_connectors/server/connector_types/security/tines/render.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { set } from 'lodash/fp';
+import { ExecutorParams } from '@kbn/actions-plugin/server/sub_action_framework/types';
+import { RenderParameterTemplates } from '@kbn/actions-plugin/server/types';
+import { SUB_ACTION } from '../../../../common/connector_types/security/tines/constants';
+
+interface Context {
+  alerts: Array<Record<string, unknown>>;
+}
+
+export const renderParameterTemplates: RenderParameterTemplates<ExecutorParams> = (
+  params,
+  variables
+) => {
+  if (params?.subAction !== SUB_ACTION.RUN) return params;
+
+  // Remove the "kibana" entry from all alerts to reduce weight, the same data can be found in other parts of the alert object.
+  const alerts = (variables.context as Context).alerts.map(({ kibana, ...alert }) => alert);
+
+  let body: string;
+  try {
+    body = JSON.stringify(set('context.alerts', alerts, variables));
```

Comment:
Just confirming that we want to send the entire `variables` field as the body and not jus the `alerts`?

![image](https://user-images.githubusercontent.com/56361221/200944014-6a4cdecb-2f6a-4a76-922c-a6b6622c767b.png)

Reply:
Also, https://github.com/elastic/kibana/pull/143505#discussion_r1017819185

Reply:
yes, this is the expected payload. we confirmed that with Tines team. 
It is the entire `variables` body with one small nuance: we clean the `kibana` entry inside the alert data, if present, to reduce some weight since this entry has a lot of duplicated information (for all items in the _context.alerts_ array).
Once the framework has an alternative to Mustache syntax, and we are able to transform/map the execution content to JSON format, instead of ndJSON, we can start using it in this connector.